### PR TITLE
Track intubated separately to Vent(ilated).

### PR DESCRIPTION
**Pull request by baryluk**
Fix TI scraper to properly report intubated, not ventilated.

One of the commits also provides improvement for bulletin URL extraction.

See also: https://github.com/openZH/covid_19/issues/215

### CODE REVIEW

**Comment by baryluk** [comment]
Does this make sense? I didn't finish adjusting `add_db_entry.py`, and left it for @metaodi to finish, as it requires establishing sqlite and csv format.

**Comment by baryluk** [comment]
`$ ./scrape_ti.sh | ./parse_scrape_output.py | ./convert_parsed_to_csv.py `


```csv
date,time,abbreviation_canton_and_fl,ncumul_tested,ncumul_conf,ncumul_hosp,ncumul_ICU,ncumul_vent,ncumul_released,ncumul_deceased,source,ncumul_ICF,ninst_ICU_intub
2020-03-28,08:00,TI,,1727,385,69,,,87,"Scraper for TI at 2020-03-28T23:56:48+01:00 using https://www4.ti.ch/dss/dsp/covid19/home/, https://www4.ti.ch/area-media/comunicati/, https://www4.ti.ch/area-media/comunicati/dettaglio-comunicato/?NEWS_ID=187538&tx_tichareamedia_comunicazioni%5Baction%5D=show&tx_tichareamedia_comunicazioni%5Bcontroller%5D=Comunicazioni&cHash=736a7f1a780ed9bdb1d8e45f8a0326ff",,60
```

(scroll to the far right)

**Comment by metaodi** [comment]
I'll look into it.

File: scrapers/parse_scrape_output.py [hunk 1]
```diff
@@ -242,6 +243,12 @@ def parse_date(d):
       except:
         warns.appent(f"ICU ({v}) not a number")
       continue
+    if k.startswith("Intubated"):
+      try:
+        intubated = int(v)
+      except:
```
Comment (cclauss) [inline scrapers/parse_scrape_output.py]: Please avoid `bare exceptions` as discussed in PEP8 and https://realpython.com/the-most-diabolical-python-antipattern
```suggestion
      except (TypeError, ValueError):
```

Comment (baryluk) [reply scrapers/parse_scrape_output.py]: @cclauss Could you take a look at https://github.com/openZH/covid_19/pull/333

**Comment by baryluk** [comment]
PTAL

```
$ ./scrape_ti.sh 
TI
Downloading: https://www4.ti.ch/dss/dsp/covid19/home/
Scraped at: 2020-03-30T22:51:55+02:00
Date and time: 30 marzo 2020, ore 8.00
Confirmed cases: 1962
Deaths: 105
Downloading: https://www4.ti.ch/area-media/comunicati/
Downloading: https://www4.ti.ch/area-media/comunicati/dettaglio-comunicato/?NEWS_ID=187546&tx_tichareamedia_comunicazioni%5Baction%5D=show&tx_tichareamedia_comunicazioni%5Bcontroller%5D=Comunicazioni&cHash=1465eace9ac68357130a7327d0c9b8f9
Scraped at: 2020-03-30T22:51:56+02:00
Date and time: 30.03.2020 ore 08:00
Hospitalized: 415
ICU: 75
Intubated: 61
$ ./scrape_ti.sh  | ./parse_scrape_output.py 
TI 2020-03-30T08:00    1962     105 OK 2020-03-30T22:52:00+02:00 # Extras: ncumul_hosp=415,ncumul_ICU=75,ninst_ICU_intub=61 # URLs: https://www4.ti.ch/dss/dsp/covid19/home/, https://www4.ti.ch/area-media/comunicati/, https://www4.ti.ch/area-media/comunicati/dettaglio-comunicato/?NEWS_ID=187546&tx_tichareamedia_comunicazioni%5Baction%5D=show&tx_tichareamedia_comunicazioni%5Bcontroller%5D=Comunicazioni&cHash=1465eace9ac68357130a7327d0c9b8f9
```

**Comment by tlorusso** [comment]
Hei @baryluk , we will store the number of intubated in the 'ncumul_vent' column from now on. A differentiation between ventilated and intubated is not necessary at the moment. The Cantons seem all to publish figures on invasive ventilation, so we can consistently collect them in the already existing 'ncumul_vent' column, without having to break the structure. A minor change needed in this PR -> rename ninst_ICU_intub into ncumul_vent. Sorry for this.. took until yesterday to get clarification.

**Comment by baryluk** [comment]
@tlorusso Ok. I will pass intubated numbers into ncumul_vent (ninst_vent actually). Thanks for clarifying and getting in touch with contacts to make it more consistent (still probably require some work and tracking to make sure it is what / how we really really want to track).

**Comment by metaodi** [comment]
> @tlorusso Ok. I will pass intubated numbers into ncumul_vent (ninst_vent actually). Thanks for clarifying and getting in touch with contacts to make it more consistent (still probably require some work and tracking to make sure it is what / how we really really want to track).

This is not done yet, right? Please let me know whan I can review this PR.

**Comment by baryluk** [comment]
@metaodi Actually, existing scraper in the repo extracts the intubated as Vent:

```
$ ./scrape_ti.sh 
TI
Downloading: https://www4.ti.ch/dss/dsp/covid19/home/
Scraped at: 2020-04-02T01:39:01+02:00
Date and time: 1 aprile 2020, ore 8.00
Confirmed cases: 2195
Deaths: 132
Downloading: https://www4.ti.ch/area-media/comunicati/
Downloading: https://www4.ti.ch/area-media/comunicati/dettaglio-comunicato/?NEWS_ID=187555&tx_tichareamedia_comunicazioni%5Baction%5D=show&tx_tichareamedia_comunicazioni%5Bcontroller%5D=Comunicazioni&cHash=843d72993004667ab731d64142591372
Scraped at: 2020-04-02T01:39:04+02:00
Date and time: 01.04.2020 ore 08:00
Hospitalized: 396
ICU: 76
Vent: 72
$ ./scrape_ti.sh  | ./parse_scrape_output.py 
TI 2020-04-01T08:00    2195     132 OK 2020-04-02T01:39:09+02:00 # Extras: ncumul_hosp=396,ncumul_ICU=76,ncumul_vent=72 # URLs: https://www4.ti.ch/dss/dsp/covid19/home/, https://www4.ti.ch/area-media/comunicati/, https://www4.ti.ch/area-media/comunicati/dettaglio-comunicato/?NEWS_ID=187555&tx_tichareamedia_comunicazioni%5Baction%5D=show&tx_tichareamedia_comunicazioni%5Bcontroller%5D=Comunicazioni&cHash=843d72993004667ab731d64142591372
$ 
```

So I think there is nothing to be done actually.

**Pull request closed without merging**